app/assets/javascripts/shared/shared.js.coffee

```diff
@@ -0,0 +1,4 @@
+window.OFNShared = angular.module("OFNShared", [
```

Comment:
Again, not my area of expertise - is this how we attach the other two modules, to the `window` object?

Reply:
This is the way we _declare_ this new module: `OFNShared`, so it can be used elsewhere.